client.go

```diff
@@ -303,13 +303,24 @@ type Client struct {
 	// ConfigureClient configures the fasthttp.HostClient.
 	ConfigureClient func(hc *HostClient) error
 
-	mLock      sync.Mutex
+	mLock      sync.RWMutex
+	mOnce      sync.Once
 	m          map[string]*HostClient
 	ms         map[string]*HostClient
 	readerPool sync.Pool
 	writerPool sync.Pool
 }
 
+// GetHostClients return the map of Http HostClient
+func (c *Client) GetHostClients() map[string]*HostClient {
+	return c.m
+}
+
+// GetHttpsHostClients return the map of Https HostClient
+func (c *Client) GetHttpsHostClients() map[string]*HostClient {
+	return c.ms
+}
```

Comment:
Why add these methods? Since `m` and `ms` are protected by `mLock` it's impossible to use the return value of these functions in a safe way?

Reply:
This is test code. I will delete it